templates/noir/.sindriignore

```diff
@@ -1,3 +1 @@
 /targets/
-Prover.toml
-Verifier.toml
```

Comment:
Semi-unrelated, but I think these are actually useful to have in the package now that we have the clone capability.